Documentation~/components.md

```diff
@@ -0,0 +1,15 @@
+# ProBuilder mesh components
+
+For a GameObject to be a ProBuilder object, it needs the following components:
+
+* [ProBuilder MeshFilter](ProBuilderMesh.md): All ProBuilder shapes have this components. It displays the **Object Size** of the ProBuilder Mesh in **X**, **Y**, and **Z**. It also allows you to generate lightmap UVs for the Mesh, and customize how ProBuilder generates them. 
+* [PolyShape (script)](polyshape.md): For custom shapes.
+* [Bezier shape (script)](bezier.md) (Experimental)
+* Mesh Renderer and Mesh Collider. These components are standard for 3D objects. For more information, refer to [GameObjects](https://docs.unity3d.com/6000.0/Documentation/Manual/GameObjects.html). 
+
```

Comment:
"needs" suggests that it *must* have *all* of these. But that is not the case. For example, not every ProBuilder object needs a Bezier shape script. Can we rewrite the introduction to this list?

Reply:
```suggestion
For a GameObject to be a ProBuilder object, it needs the following components:

* [ProBuilder MeshFilter](ProBuilderMesh.md): All ProBuilder shapes have this components. It displays the **Object Size** of the ProBuilder Mesh in **X**, **Y**, and **Z**. It also allows you to generate lightmap UVs for the Mesh, and customize how ProBuilder generates them. 
* [PolyShape (script)](polyshape.md): For custom shapes.
* Mesh Renderer and Mesh Collider. These components are standard for 3D objects. For more information, refer to [GameObjects](https://docs.unity3d.com/6000.0/Documentation/Manual/GameObjects.html). 

```